packages/mlocate.rb

```diff
@@ -2,35 +2,39 @@
 
 class Mlocate < Package
   description 'mlocate is a locate/updatedb implementation.'
-  homepage 'https://pagure.io/mlocate'
-  version '0.26-1'
+  homepage 'https://pagure.io/mlocate/'
+  version '0.26-2'
   license 'GPL-2'
   compatibility 'all'
   source_url 'https://releases.pagure.org/mlocate/mlocate-0.26.tar.xz'
   source_sha256 '3063df79fe198fb9618e180c54baf3105b33d88fe602ff2d8570aaf944f1263e'
 
-  binary_url ({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mlocate/0.26-1_armv7l/mlocate-0.26-1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mlocate/0.26-1_armv7l/mlocate-0.26-1-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mlocate/0.26-1_i686/mlocate-0.26-1-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/mlocate/0.26-1_x86_64/mlocate-0.26-1-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '69e02f44a498cd38a8f4bc2e3410f934506426d670664c402a09bfb34d51418c',
-     armv7l: '69e02f44a498cd38a8f4bc2e3410f934506426d670664c402a09bfb34d51418c',
-       i686: '74acf4453947b3570f68c43e485cb8e9f2da688e4a50336d593bbba1b6ebd2ee',
-     x86_64: 'f943fe00ad4666f5a9c7bf12d710cd8f2e136a33ad94568863b92dd45526d7eb',
-  })
+  def self.patch
+      @updatedb_wrapper = <<~EOF
+      #!/bin/bash -e
+      if [ "${EUID}" = 0 ]; then
+        echo "updatedb should not be run as root on ChromeOS."
```

Comment:
Why not?

Reply:
"to keep ownership consistent throughout `/usr/local` in accordance with (currently unwritten) Chromebrew standards, `updatedb` cannot be run as root"

Reply:
Minus that consistency, should someone run `updatedb` as root, and then attempt to run it as a normal user, they will receive an error. Also, running as a normal user is a very slight security improvement, as it doesn't save `/proc` or `/sys` in the database.

Reply:
How about instead of restricting it altogether, reset the ownership back afterwards?  I think this restriction is too limiting.

Reply:
We could, but that would require the use of `sudo`, as `chown` seems to require `sudo` all the time. So running `updatedb` would then require a password.

Reply:
@satmandu, @supechicken second opinions?

Reply:
I don't think sudo should be a requirement.  You should be able to run `updatedb` or `sudo updatedb` separately.